### PR TITLE
Add BCV

### PR DESCRIPTION
**Pull request by Goooler**
https://github.com/Kotlin/binary-compatibility-validator

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -36,3 +36,4 @@ jetbrains-compose = { id = "org.jetbrains.compose", version.ref = "jetbrains-com
 ben-manes-versions = "com.github.ben-manes.versions:0.51.0"
 nexus-publish = "io.github.gradle-nexus.publish-plugin:2.0.0"
 kotlinter = "org.jmailen.kotlinter:4.4.1"
+bcv = "org.jetbrains.kotlinx.binary-compatibility-validator:0.15.1"
```
Comment (svenjacobs) [inline gradle/libs.versions.toml:39]: I'd like to be a bit more explicit about the name. Maybe `binary-compat-validator`?

**Review by svenjacobs — CHANGES_REQUESTED** [review]
Thanks for the PR! I just have one remark.

File: gradle/libs.versions.toml (hunk 1 above)
Comment (Goooler) [reply gradle/libs.versions.toml:39]: According to it's group, it might be `jetbrains-bcv` or `kotlinx-bcv`. Anyway, I can rename it to others you prefer.

Comment (svenjacobs) [reply gradle/libs.versions.toml:39]: As you can see with the other plugins, I don't have a strict naming scheme. It should just be understandable. I like the name I proposed before 😉

Comment (Goooler) [inline gradle/libs.versions.toml:39]:
```suggestion
binary-compat-validator = "org.jetbrains.kotlinx.binary-compatibility-validator:0.15.1"
```

File: build.gradle.kts [hunk 2]
```diff
@@ -14,6 +14,7 @@ plugins {
 	alias(libs.plugins.nexus.publish)
 	alias(libs.plugins.ben.manes.versions)
 	alias(libs.plugins.kotlinter)
+	alias(libs.plugins.bcv)
```
Comment (Goooler) [inline build.gradle.kts:17]:
```suggestion
	alias(libs.plugins.binary.compat.validator)
```

**Review by svenjacobs — APPROVED** [review]

**Review by svenjacobs — APPROVED** [review]

**Pull request merged**